templates/zerver/help/collapse-a-message.md

```diff
@@ -0,0 +1,8 @@
+## Collapse a message
+
+Zulip allows you to collapse messages for enhanced message viewing abilities.
```

Comment:
Here I would probably change " for enhanced message viewing abilities" to "to reduce clutter."